pkgs/applications/blockchains/polkadot/default.nix

```diff
@@ -7,16 +7,16 @@
 }:
 rustPlatform.buildRustPackage rec {
   pname = "polkadot";
-  version = "0.9.9-1";
+  version = "0.9.10";
 
   src = fetchFromGitHub {
     owner = "paritytech";
     repo = "polkadot";
     rev = "v${version}";
-    sha256 = "sha256-EmnrwBMHb9jpEZAG393yyMaFRRQJ6YYDRvsp+ATT7MY=";
+    sha256 = "sha256-1iBA7R63g0UOuCmnFONc9j/7jfcHA54jJ327AL/aPIE=";
   };
 
-  cargoHash = "sha256-WzsaUrqe7F4x+ShqG14kq78MTSWIxIMRa3pdr3RXrwk=";
+  cargoHash = "sha256:1ysbyv323qmpswbnd2lvisrwnk30l4zvcidh866nrk6b6jqjag42";
```

Comment:
Same with the prefix as above. I might consider to use the base64 encoded version of the hash here too, i.e. `gjwlsTTLzGyNQbBFtj+hYEzLs46bimYX17fiIcb2S/s=`.

Reply:
How can I get base64-encoded version? I get non-base64 encoded version from builder error when I put "sha256:${lib.fakeSha256}"

Reply:
```suggestion
  cargoHash = "1ysbyv323qmpswbnd2lvisrwnk30l4zvcidh866nrk6b6jqjag42";
```

Reply:
Use nix hash to convert them.